tests/test_circuit.py

```diff
@@ -16,7 +15,7 @@
 
 def test_iqp_ansatz():
     diagram = (Word('Alice', N) @ Word('runs', N >> S) >>
-               Cup(N, N.r) @ Id(S))
+               Cup(N, N.r) @ S)
```

Comment:
I think we should not have these changes (i.e. all of the form `Id(X)` to `X`) for this PR because they're not necessary and add noise to the large number of changes we have to review. Future PR

Reply:
I feel that undoing those changes takes more time than quickly skimming over the according lines.